ui/entry-index.js

```diff
@@ -35,6 +35,7 @@ require('ngreact');
 require('jquery.scrollto');
 require('json-schema-defaults');
 require('./vendor/resizer.js');
+require('./vendor/jquery-migrate-3.0.0.js');
```

Comment:
Yeah, those commits were just so I could debug without having to add these each time.  :)  I'll dump the commits before merge.